openc3/lib/openc3/api/settings_api.rb

```diff
@@ -30,7 +30,8 @@ module Api
                        'get_all_settings',
                        'get_setting',
                        'get_settings',
-                       'save_setting'
+                       'set_setting',
+                       'save_setting' # DEPRECATED
```

Comment:
I thought it made sense to have a `set` for the existing `get` rather than save.